system.ttl

```diff
@@ -0,0 +1,13 @@
+@prefix org.dice.factcheckbenchmark.benchmark.rdf: <http://www.w3.org/1999/02/22-org.dice.factcheckbenchmark.benchmark.rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/org.dice.factcheckbenchmark.benchmark.rdf-schema#> .
+@prefix ex: <http://example.org/> .
+@prefix hobbit: <http://w3id.org/hobbit/vocab#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+
+<http://project-hobbit.eu/factcheck-benchmark/factcheck>	a	hobbit:SystemInstance;
```

Comment:
Can you please use a different URI, e.g., `http://dice.cs.uni-paderborn.de/factcheck` or something like that.